pkg/features/kube_features.go

```diff
@@ -849,7 +849,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	// inherited features from generic apiserver, relisted here to get a conflict if it is changed
 	// unintentionally on either side:
-	genericfeatures.StreamingProxyRedirects: {Default: true, PreRelease: featuregate.Deprecated},
+	genericfeatures.StreamingProxyRedirects: {Default: false, PreRelease: featuregate.Deprecated}, // remove in 1.24
 	genericfeatures.ValidateProxyRedirects:  {Default: true, PreRelease: featuregate.Beta},
```

Comment:
Can we also mark ValidateProxyRedirects as deprecated? We don't need to change the default, but this will go away when the `StreamingProxyRedirects` code is removed.

Reply:
make sense. 

> 	// ValidateProxyRedirects controls whether the apiserver should validate that redirects are only
> 	// followed to the same host. Only used if StreamingProxyRedirects is enabled.

Updated.